build/environment.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+export PYTHONPATH=$HOME/Insight/Vulnerable_Buildings/Modules:$PYTHONPATH
```

Comment:
See comment above - use something that would work for anyone using this code. 
E.g. put this at the top level of your repo. 
```
export PYTHONPATH=$PYTHONPATH:`pwd`
```
Then for imports use `import Modules` at the top of your python file. 

Ideally, don't use this path hacking at all.